docs/gambit/mutation-verifier.md

```diff
@@ -35,129 +34,193 @@ If you are on Linux, you will need pip version 20.3 or above.
-| `--dump_csv`                   | specify a CSV file to write the verification report. Much of this content is in the JSON report as well, but this is intended to be an alternative representation of the data                                                              |
-| `--collect_file`               | specify the collect file from which to run in asynchronous mode                                                       |
-| `--sync`                       | enable synchronous execution                                                                                          |
-| `--max_timeout_attempts_count` | specify the maximum number of times a web request is attempted                                                        |
-| `--request_timeout`            | specify the length in seconds for a web request timeout                                                               |
-| `--poll_timeout`               | specify the number of minutes to poll a task in sync mode before giving up. Polling is possible even after the timeout with another call to `certoraMutate`     |
+| `--mutation_conf`              | Specify the configuration file for mutant generation                                                                  |
+| `--orig_run`                   | Specify the {ref}`orig-verification-link`                                                            |
+| `--orig_run_dir`               | Specify the folder where the files will be downloaded from the {ref}`orig-verification-link` |
+| `--prover_conf`                | Specify the {ref}`prover-conf` file for verifying mutants                                                           |
+| `--prover_version`             | Specify the `certoraRun` version for verification. If not specified, the installed `certoraRun` version is used by default.        |
+| `--msg`                        | Add a message to identify the `certoraMutate` run |
+| `--gambit_only`                | Stop processing after generating mutations |
+| `--dump_link`                  | Specify a text file to write the UI report link                                                                       |
+| `--dump_failed_collects`       | Specify a log file to capture mutant collection failures |
+| `--debug`                      | Show additional logging information during execution                                                                  |
 
 ## Troubleshooting
 
-At the moment, there are a few ways in which `certoraMutate` can fail. Here are some suggestions on how to troubleshoot when that happens. We are actively working on mitigating them.
+At the moment, there are a few ways in which `certoraMutate` can fail. 
+Here are some suggestions on how to troubleshoot when that happens. 
+We are actively working on mitigating them.
 
-- Sometimes it might be useful to first run `gambit` without going through `certoraMutate`.
+- Sometimes it is easier to understand the problem by running `gambit` directly.
```

Comment:
now that we have the gambit-only mode and also allow passing gconf files, should we mention that instead of this? The advantage would be that the user only needs to run one tool.

Reply:
Good point. It would still obscure some Gambit error messages potentially, but I think that after Rahav's fix we got this covered. Changed.